src/components/Policy/EmptyState/Section.tsx

```diff
@@ -28,7 +28,6 @@ export const EmptyStateSection: React.FunctionComponent<EmptyStateSectionProps>
         className={ `${emptyStateClassName} ${props.className ? props.className : ''} ` }
         { ...getOuiaProps('Policy/EmptyState', props) }
     >
-        { props.icon && <EmptyStateIcon icon={ props.icon } color={ props.iconColor } /> }
```

Comment:
I think we will still need line to show the icon in other empty states.

Reply:
You need to find where `EmptyStateSection` is being used and remove the `icon` and `iconColor`  in the places where we need to remove (referred by the jira)